src/routes/newUser.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import { User } from '../models';
+import * as users from '../controllers/users';
+
+const router = express.Router();
+
+router.get('/', function (req, res) {
+  res.render('newUser');
+});
+
+router.post('/', function (req, res) {
+  const newUser = new User(
+    2,
+    req.body.username,
+    req.body.password,
+    req.body.email,
+    req.body.displayName,
+    3,
+  );
+
+  users.pushUser(newUser);
```

Comment:
To be clear this wont be persisted between server sessions?

Reply:
Correct. When I restarted the server it lost all accounts I had pushed. If we want to persist beyond that we have to have a file off the server somewhere to hold are accounts instead. Refactoring for that should be pretty easy though.